src/Mado/QueryBundle/Component/Meta/Exceptions/UnInitializedQueryBuilderException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Mado\QueryBundle\Component\Meta\Exceptions;
+use Throwable;
+
+/**
+ * @since Class available since Release 2.2.0
```

Comment:
Version is 2.2.3. What do you think about `since` tag. I found that is not so useful I was thinking. Maybe third number is not so useful here. What's your opinion?

Reply:
I think that third number could be removed, I'll do It

Reply:
👍